clang/test/CodeGen/excess-embed-data.c

```diff
@@ -0,0 +1,55 @@
+// RUN: %clang_cc1 -std=c23 -emit-llvm %s -o - | FileCheck %s
```

Comment:
We probably need to specify a target here (linux x64) as we rely on the size of int